packages/squiggle-lang/src/reducer/lambda.ts

```diff
@@ -128,22 +129,26 @@ export class UserDefinedLambda extends Lambda {
     return `lambda(${this._getParameterNames().join(",")}=>internal code)`;
   }
 
-  paramCounts() {
+  parameterCounts() {
     return [this.parameters.length];
   }
+
+  parameterCountString() {
+    return this.parameters.length.toString();
+  }
 }
 
 // Stdlib functions (everything in FunctionRegistry) are instances of this class.
-export class BuiltinLambda extends Lambda {
-  type = "BuiltinLambda";
-  _definitions: FnDefinition[];
+export class BuiltinLambda extends BaseLambda {
+  readonly type = "BuiltinLambda";
+  _signatures: FnDefinition[];
```

Comment:
I guess find-and-replace went wrong? This should be `_definitions`, not `_signatures`.

Reply:
Sorry - I thought you wanted me to replace all of these with _signatures

Reply:
No, my point was that there's "definition" (something callable) and there's a "signature" (just a metadata for a definition, with parameter names and domains), and those are two different types which should be named accordingly.